### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by Jcambass** [diacode/togglex#4]
Looks like this repo isn't maintained anymore... that's quite sad because the toggl_api_docs still link to it.

Since there are 2 usable / needed pull requests we may transfer the repo to one that is interested in at least somewhat maintaining it?

What do you guys think? @stringlytyped  @komichar @diacode

**Comment by stringlytyped** [comment]
When I submitted my pull request (#3), I was fairly certain I wouldn't get a response because it was quite obvious, even at that point, that this project was dead. (I submitted it anyway so that anyone that might be running into the same problem would be able to find a documented solution.)

I would like for this project to be more actively maintained, definitely. But, at the same time, I am not using it for anything super mission critical.

Interestingly, there doesn't appear to be any mention of the license under which the code is released. So, one would probably need permission from @diacode to fork it.

@bbay @cabify @hopsor @javiercr
